tonic_validate/metrics/context_length_metric.py

```diff
@@ -18,7 +18,8 @@ def __init__(
         max_length: Optional[int] = None,
     ):
         """
-        Create a metric that checks if each item in the context list is within a given length.
+        Create a binary metric that checks if each item in the context list is within a given length.
```

Comment:
Creates a binary metric that checks whether each item in the context list is within a given length.